### ISSUE TITLE
Difficult to see comments highlighted in diff mode

### ISSUE THREAD

**Issue by AdamVig** [emroussel/atomize#2]
I just switched over to Atomize and I like it so far, but I ran into an issue with the contrast between the color of green used for diffs and the color of gray used for comments.

![image](https://user-images.githubusercontent.com/5354752/54546545-4d1b8e00-497a-11e9-8b9b-6c60ec6c8e00.png)

As you can see in the image, the comment on the line that is highlighted in green is difficult to distinguish.

I would make a pull request, but I don't know where to begin. Can this color be changed to fix this issue without any unintended side effects?

**edit**: looks like the relevant colors to customize are under the key [`diffEditor`](https://code.visualstudio.com/api/references/theme-color#diff-editor-colors).

**Comment by emroussel** [comment]
Thanks for reporting this! Yes, it looks like it can customized using `diffEditor`. It might be a bit challenging to find a color that provides enough contrast with comments and the other colors, but I'll play with it a bit and see what I can do.

**Comment by emroussel** [comment]
@AdamVig I made the color of diffs darker, which makes comments a little easier to read. Still not the greatest contrast, but let me know if that solves your issue. The update should be available in the marketplace soon under `1.3.0` (I also added a few other improvements when using git).

<img width="320" alt="Screen Shot 2019-03-19 at 19 58 04" src="https://user-images.githubusercontent.com/12487270/54688244-a6f39380-4af3-11e9-878f-88db1cd570a9.png">
